### ISSUE TITLE
'isArray' does not exist in type 'ParamOptions'

### ISSUE THREAD

**Issue by sshd123** [typestack/routing-controllers#831]
i use query param(array type) on controller,but when i debug in vscode, it raise error.
query is :`@QueryParam("models", { isArray: true}) models: string[]`
error is: Object literal may only specify known properties, and 'isArray' does not exist in type 'ParamOptions'

**Comment by faridanthony** [comment]
Here is the interface, it doesn't include isArray. Where do you see that is supported?

```
export interface ParamOptions {
    /**
     * If set to true then parameter will be required.
     * If user performs a request and required parameter is not in a request then routing-controllers will throw an error.
     */
    required?: boolean;
    /**
     * If set to true then parameter will be parsed to json.
     * Parsing is automatically done if parameter type is a class type.
     */
    parse?: boolean;
    /**
     * Class transform options used to perform plainToClass operation.
     */
    transform?: ClassTransformOptions;
    /**
     * If true, class-validator will be used to validate param object.
     * If validation options are given then class-validator will perform validation with given options.
     */
    validate?: boolean | ValidatorOptions;
    /**
     * Explicitly set type which should be used for param to perform transformation.
     */
    type?: any;
}
```

**Comment by attilaorosz** [comment]
Closing as stale.

**Comment by talbSE** [comment]
this wasnt released yet, is there any plan for releasing 0.10 rc1 ?  https://github.com/typestack/routing-controllers/issues/798
or backward compatible this to 0.9 for better array support - https://github.com/typestack/routing-controllers/pull/677
